jest.config.js

```diff
@@ -1,16 +1,24 @@
 'use strict';
 
-const { compose, baseConfig, withRTL } = require('@moxy/jest-config');
+const { compose, baseConfig } = require('@moxy/jest-config-base');
+const withWeb = require('@moxy/jest-config-web');
+const { withRTL } = require('@moxy/jest-config-testing-library');
```

Comment:
Can we change this to a single line import?

```suggestion
const { compose, baseConfig, withWeb, withRTL } = require('@moxy/jest-config');
```